docker-compose.yml

```diff
@@ -1,80 +1,74 @@
 version: "3"
 services:
-  zookeeper:
-    image: 'bitnami/zookeeper:latest'
-    ports:
-      - ${ZK_PORT}:${ZK_PORT}
-    environment:
-      #- ALLOW_ANONYMOUS_LOGIN=yes
-      #- ZOO_HEAP_SIZE=512
-      - ZOO_ENABLE_AUTH=yes
-      - ZOO_SERVER_USERS=${ZOO_SERVER_USERS}
-      - ZOO_SERVER_PASSWORDS=${ZOO_SERVER_PASSWORDS}
-      - ZOO_CLIENT_USER=${ZOO_CLIENT_USER}
-      - ZOO_CLIENT_PASSWORD=${ZOO_CLIENT_PASSWORD}
-      - ZOO_TLS_CLIENT_ENABLE=yes
-      - ZOO_TLS_CLIENT_KEYSTORE_FILE=${ZOO_TLS_CLIENT_KEYSTORE_FILE}
-      - ZOO_TLS_CLIENT_KEYSTORE_PASSWORD=${ZOO_TLS_CLIENT_KEYSTORE_PASSWORD}
-      - ZOO_TLS_CLIENT_TRUSTSTORE_FILE=${ZOO_TLS_CLIENT_TRUSTSTORE_FILE}
-      - ZOO_TLS_CLIENT_TRUSTSTORE_PASSWORD=${ZOO_TLS_CLIENT_TRUSTSTORE_PASSWORD}
-    volumes:
-      - ${KAFKA_ZOOKEEPER_CERTS_PATH}:/opt/bitnami/kafka/conf/certs
-    networks:
-      - tombolo-network
+  # zookeeper:
```

Comment:
Does it make sense to move this to a docker-compose-with-kafka.yml file and remove the commented out code services from docker-compose.yml?